### DISCUSSION TITLE
Improving video quality

### DISCUSSION THREAD

**Discussion by monocilindro** [tylermorganwall/rayshader#228] [Ideas]
The video recordings generated using the "render_movie" command have a 420p resolution. That quality could be not good enough for certain purposes. It would be nice having the chance to improve it

**Comment by tylermorganwall** [comment]
The resolution is taken directly from the rgl window size: set the `windowsize` argument in `plot_3d()` to increase the resolution to whatever value you require.

**Reply by monocilindro** [reply]
It seems that the windowsize argument is only applied to the onscreen rendering and the snapshots but not to the videos. I cannot get any resolution higher than 420p, even though when the "windowside" is set to 1080p or more.

**Comment by tylermorganwall** [comment]
Looks like there was a bug reversing the video dimension order, so non-square videos used the wrong resolution—try the latest github version and it should be fixed.
